test/models/test_models.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*_
+#
+# Copyright (c) 2020, Pureport, Inc.
+# All Rights Reserved
+
+import pytest
+import random
+from pureport import models
+from ..utils import utils
+from .test_helpers import ModelData
+models.make()
```

Comment:
This needs to be mocked out and the openapi.json file loaded from `test/openapi.json`

Reply:
Done.